indico/modules/events/templates/display/indico/lecture.html

```diff
@@ -14,11 +14,21 @@
     {% block header %}
         <div class="event-header event-header-lecture {%- if not has_subheader %} round-bottom-corners{% endif %}">
             {% if not g.static_site %}
-                <a class="lecture-category" href="{{ url_for('categories.display', event.category) }}">
-                    {{ category }}
-                </a>
+                {% if category %}
+                    <a class="lecture-category" href="{{ url_for('categories.display', event.category) }}">
+                        {{ category }}
+                    </a>
+                {% else %}
+                    <a class="lecture-category" href="{{ url_for('event_management.settings', event) }}">
+                        {% trans %}Unlisted event{% endtrans %}
+                    </a>
+                {% endif %}
             {% else %}
-                <div>{{ category }}</div>
+                {% if category %}
+                    <div>{{ category }}</div>
+                {% else %}
+                    <div>{% trans %}Unlisted event{% endtrans %}</div>
```

Comment:
I wouldn't include that when building a static site... even during normal usage I wonder if we shouldn't just hide the category altogether if there isn't one..

Reply:
I thought it would be a good place to link to the event management view, as usually it will be the creator who visits the display view. But I'm fine either way.

Reply:
Hm... there's already the button pointing to the management area anyway..

Reply:
Yup, and it's pretty close too. I'll remove it.